cms/static/js/views/settings/main.js

```diff
@@ -164,7 +209,26 @@ var DetailsView = ValidatingView.extend({
         $(e.currentTarget).attr('title', currentTimeText);
     },
     updateModel: function(event) {
+        var value;
+        var index = event.currentTarget.getAttribute('data-index');
         switch (event.currentTarget.id) {
+        case 'course-learning-info-' + index:
```

Comment:
Sorry, I could not get your point here. Are you talking about `learning_info.js` view ? If Yes then I am not updating model attributes in `learning_info.js` and also not in `instructor_info.js`.